commands/list_clusters.go

```diff
@@ -43,9 +44,14 @@ func checkListClusters(cmd *cobra.Command, args []string) error {
 
 // list all clusters the user has access to
 func listClusters(cmd *cobra.Command, args []string) {
-	client := gsclientgen.NewDefaultApiWithBasePath(cmdAPIEndpoint)
+	clientConfig := client.Configuration{
+		Endpoint:  cmdAPIEndpoint,
+		Timeout:   3 * time.Second,
```

Comment:
I want to fail fast in situations where multiple GET calls are made in sequence. E. g. when listing clusters, we first fetch all the organizations (memberships). Then we look for clusters in each org. This failing after one minute (as it happened just recently with flapping API behaviour) is just a really bad waste of time for the end user. I'd rather fail after a few seconds and let the user retry.

In cases of transactions (POST, PUT, PATCH) I'd like to tolerate longer timeouts, as we expect a little bit more work to happen on the other end, and retrying might not be as easy (unless transactions are implemented for that route).

The next step of course will be doing the retrying automatically, so the user doesn't have to bother at all in most cases.

It's a tiny step, but I see is as a little piece in a bigger resiliance picture.